queries/r/locals.scm

```diff
@@ -1,11 +1,11 @@
 ; locals.scm
 
-(function_definition) @local.scope
+(function_definition) @scope
 
-(formal_parameters (identifier) @local.definition)
+(formal_parameters (identifier) @definition)
```

Comment:
`@definition.parameter`?

Reply:
Yea, probably. Thanks.

Off topic observation: having many capture groups [in CONTRIBUTING.md](https://github.com/nvim-treesitter/nvim-treesitter/blob/master/CONTRIBUTING.md#locals) without explanations of what they actually mean creates a need for guessing.